StripePayments/StripePayments/Source/API Bindings/Models/Shared/STPFormEncodableClientAttributionMetadata.swift

```diff
@@ -1,15 +1,14 @@
 //
-//  STPClientAttributionMetadata+Payments.swift
+//  STPFormEncodableClientAttributionMetadata.swift
 //  StripePayments
 //
-//  Created by Joyce Qin on 8/4/25.
+//  Created by Joyce Qin on 10/02/25.
 //
 
 import Foundation
 @_spi(STP) import StripeCore
 
-// MARK: - STPFormEncodable
-extension STPClientAttributionMetadata: STPFormEncodable {
+@_spi(STP) public class STPFormEncodableClientAttributionMetadata: STPClientAttributionMetadata, STPFormEncodable {
```

Comment:
I thought that we wanted to avoid inheritance across modules that need to obj-c compatibility?